src/support_code_library_builder/world.ts

```diff
@@ -1,10 +1,17 @@
 import { IAttachment } from '../runtime/attachment_manager'
 
+export type IAttachFunction = (attachment: IAttachment) => void
```

Comment:
This should probably live in `src/runtime/attachment_manager/index.ts`

Reply:
Sure, moved